cmake/Modules/KatanaBuildCommon.cmake

```diff
@@ -631,3 +631,94 @@ file(CREATE_LINK
     )
   endif()
 endfunction()
+
+# find_c_library finds a named library and populates a CMake target with the
+# appropriate build options for use with target_link_libraries.
+#
+# It tries the following methods, in order:
+#
+# 1. find_package(<name>): Look for a cmake module for <name>
+# 2. pkg_check_modules(<name>): Look for a pkg-config config for <name>
+# 3. find_library(<name>): Look for lib<name> and MAIN_HEADER in path
+#
+# The exact paths searched by find_c_library varies based on the method and
+# version of CMake.
+#
+# If find_c_library fails to find a library, possible ways to adjust the search
+# path of find_c_library, find_package, pkg_check_modules, etc. are
+#
+# 1. Add the directory the environment variable PATH
+# 2. Add the directory to the CMake variable CMAKE_PREFIX_PATH
+# 3. Add the directory to the CMake variable ${LIBRARY_NAME}_ROOT
+function(find_c_library)
```

Comment:
I don't think this is used anywhere in this PR (anymore).

Reply:
It isn't, but I thought it was useful.  I'll take it out.